src/gui/attributetable/qgsdualview.cpp

```diff
@@ -649,7 +660,7 @@ void QgsDualView::setCurrentEditSelection( const QgsFeatureIds &fids )
 
 bool QgsDualView::saveEditChanges()
```

Comment:
@nirvn this method doesn't seem to be used anymore.
Shall we drop (deprecate) it?